components/layout/display_list/webrender_helpers.rs

```diff
@@ -209,12 +211,20 @@ impl WebRenderDisplayItemConverter for DisplayItem {
                                 stacking_context.perspective.is_some()
                         );
```

Comment:
I'd remove the assertion since it's now a release assertion effectively right below.